CommunicationLayer/CommunicationContainer.cpp

```diff
@@ -6,15 +6,26 @@
 #include "CommDeviceControl/CommDeviceManager.h"
 #include "CommDeviceControl/UdpConnectionService.h"
 #include "JsonReceiver/JsonReceiver.h"
+#include "../BusinessLayer/BusinessContainer.h"
 
 class CommunicationContainerPrivate
 {
 public:
-    CommunicationContainerPrivate()
+    CommunicationContainerPrivate(BusinessContainer& businessContainer)
         : udpConnectionService_(udpSocket_)
         , commDeviceManager_(udpSocket_)
         , connectionController_(udpConnectionService_)
-        , jsonReceiver_(commDeviceManager_)
+        , jsonReceiver_(commDeviceManager_
+                        , businessContainer.batteryPopulator()
```

Comment:
We only format inline construction like this, commas should go after each line for large function calls.